process-dir.js

```diff
@@ -224,6 +236,29 @@ async function process (dir) {
   }))
 }
 
+async function removeEmptyDirectories (dir) {
+  let fileStats = await lstat(dir)
+
+  if (!fileStats.isDirectory()) {
+    return
+  }
+
+  let filesAfter = await readdir(dir)
```

Comment:
It is not files**After** and not **files**.

Let use more compact names. `before` and `after` will be enough here.

Reply:
`before` **and** `after`? There is one variable here. Do I need to add another one?